### ISSUE TITLE
Tests throws error when rendered output has new lines

### ISSUE THREAD

**Issue by michalg42** [macku/jest-puppe-shots#5]
Hello
There is a problem when rendered output has new lines. Most of the time that's not a problem, but svg elements can have new lines in path elements.
In my case that was:
```
<svg>
<path d={`M150 0 
L75 200 
L225 200 Z`} />
</svg>
```
The problem is in this fragment (lib/renderer/common.js , line 1):
```
const wrapReactHtmlOutput = html => `document.body.innerHTML = '${html.replace("'", "\\'")}'`;
```
all you have to do is to change it so it uses template literals:
```
const wrapReactHtmlOutput = html => `document.body.innerHTML = \`${html.replace("'", "\\'")}\``;
```
also now I'm not sure if this replace is even needed, as variable html isn't surrounded by it anymore.

**Comment by macku** [comment]
Thanks, @michalg42, I will try to fix the problem in upcoming days.

**Comment by macku** [comment]
Fixed by #6
